src/htrun/host_tests_runner/host_test_default.py

```diff
@@ -130,7 +139,10 @@ def run_test(self):
         dut_event_queue = Queue()  # Events from host to DUT {k;v}
 
         def callback__notify_prn(key, value, timestamp):
-            """! Handles __norify_prn. Prints all lines in separate log line """
+            """Handle __norify_prn.
+
+            Prints all lines in separate log line
```

Comment:
```diff
- Prints all lines in separate log line
+ Print all lines in a separate log line.
```

Reply:
Also: "__norify_prn" heh. I think that should be "__notify_prn".

Reply:
xorify